index.bs

```diff
@@ -577,7 +577,7 @@ A {{WebTransport}} object has the following internal slots.
   </tr>
   <tr>
    <td><dfn>\[[ReceiveStreams]]</dfn>
-   <td class="non-normative">An [=ordered set=] of {{ReceiveStream}} objects owned by this
+   <td class="non-normative">An [=ordered set=] of {{ReceiveStreams}} owned by this
```

Comment:
This isn't linkifying correctly. Probably we need to write {{ReceiveStream}}s instead. Also below.

Reply:
Done.